### ISSUE TITLE
Unable to upload svg with 3.2.5

### ISSUE THREAD

**Issue by donbowman** [udx/wp-stateless#687]
I've been using wp-stateless with svg for some time. the latest (3.2.5) version, i get this error below when upload svg to media.
This is in wp_unique_filename() function

wp 6.3.2
php 8.2.11

```
PHP Warning:  file_exists(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 2608;
PHP message: PHP Warning:  chmod(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-admin/includes/file.php on line 1043;
PHP message: PHP Warning:  file_exists(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-admin/includes/image.php on line 742;
PHP message: PHP Warning:  exif_imagetype(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3286;
PHP message: PHP Warning:  file_get_contents(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3309;
PHP message: PHP Warning:  exif_imagetype(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3286;
PHP message: PHP Warning:  file_get_contents(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3309;
PHP message: PHP Warning:  exif_imagetype(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3286;
PHP message: PHP Warning:  file_get_contents(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3309;
PHP message: PHP Warning:  file_exists(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-admin/includes/image.php on line 651;
PHP message: PHP Warning:  file_exists(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3527;
PHP message: PHP Warning:  filesize(): Unable to find the wrapper &quot;gs&quot; - did you forget to enable it when you configured PHP? in /app/wp-includes/functions.php on line 3527" while reading response header from upstream, client: 10.16.5.249, server: _, request: "POST /wp-admin/async-upload.php HTTP/1.1", upstream: "fastcgi://unix:/run/php/php-fpm.sock:", host: "www.agilicus.com", referrer: "https://www.agilicus.com/wp-admin/post.php?post=421800&action=edit"
```

**Comment by ewsopp** [comment]
Thanks for the report. We will aim to include this in the next release (v4). Hopefully, by next week some time.

**Comment by donbowman** [comment]
Any comments on this? it makes using wp 6.4 quite difficult since only one of the gallery modes works.

**Comment by ewsopp** [comment]
It will be included in the next release.

**Comment by planv** [comment]
Hello @donbowman 

The fix was released with version `3.3.0`.
Please let us know how it works for you.

**Comment by donbowman** [comment]
on first try it indeed fixes the issue, thanks very much.
